examples/confluent-example/confluent_example/serializers.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict
+import aiokafka
+
+from schema_registry.serializers import AsyncAvroMessageSerializer
+
+
+class AvroSerializer(AsyncAvroMessageSerializer):
```

Comment:
I'm starting to wonder if some of this serializers should come with kstream itself

Reply:
Good question!  Maybe for now we can include in `kstreams` `serializers` that can be created with `native python modules` like `json` and `pickle`. If end users demands other `serializers` like `avro`, we can add them.

Reply:
For now we are fine I think, but we have to keep an eye if a pattern emerges from the examples